src/properties/table-cell-properties.ts

```diff
@@ -40,6 +40,10 @@ export type TableCellProperties = {
 		 */
 		tr2bl?: null | Border<LineBorderType>;
 	};
+	/**
+	 * The vertical alignment of this cell.
+	 */
+	vAlign?: null | 'bottom' | 'center' | 'top';
```

Comment:
This property could be more clearly named as `verticalAlignment`;

- `v` stands for `vertical`, not `velociraptor`. There is no need to abbreviate anything.
- `alignment` (rather than `align`) would be more consistent with the `alignment` property in `ParagraphProperties`

Reply:
Fixed!